.gitignore

```
@@ -42,6 +43,8 @@
 !/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**/*.svg
 !/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**/*.png
 !/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**/*.jpg
+!/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**/*.asm
+!/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/popcnt_32bit/**
```

Wie in #80 vorgeschlagen, könnte der extra-Ordner komplett freigegeben werden, also so hier, wenn ich mich nicht irre:

```suggestion
!/[A-Z]*[A-Z]-[A-Z]*[A-Z]/extra/**
```

(das sollte ja dann auch Unterordner erlauben)